ads-txt.php

```diff
@@ -31,7 +31,16 @@ function tenup_display_ads_txt() {
 		if ( ! empty( $post_id ) ) {
 			$post = get_post( $post_id );
 			header( 'Content-Type: text/plain' );
-			echo esc_html( $post->post_content );
+			$adstxt = esc_html( $post->post_content );
+
+			/**
+			 * Filter the ads.txt content.
+			 *
+			 * @since x.x.x
```

Comment:
This seems good for 1.2.0
```suggestion
			 * @since 1.2.0
```